### ISSUE TITLE
Implement GOG wrapper or/and compatiblity

### ISSUE THREAD

**Issue by Tomasak** [VFansss/mgs2-v-s-fix#131]
GOG version lacks only two things - widescreen fix and Xinput plus
Everything else works 
![mgs2_sse Screenshot 2020 09 25 - 19 24 48 96](https://user-images.githubusercontent.com/11007651/94297765-5325fe00-ff65-11ea-92dc-b56846b88211.png)
graphical effects

![obrazek](https://user-images.githubusercontent.com/11007651/94297784-5ae5a280-ff65-11ea-96f8-8752c1933ecd.png)
and even the projectors

**Comment by xatornet** [comment]
On my system, I've just deleted d3d8.dll, dxcfg.exe and dxcfg.ini from the game's bin folder, which are the GOG d3d8 wrapper files, and everything seems to work fine. Give it a try.

**Comment by Tomasak** [comment]
but GOG wrappers are superior to which V fannss currently use.
Resolution scaling, MSAA, AF all in it already, no need for SMAA injectors, no need for quartz.dll

**Comment by theDarkFrenchy** [comment]
good to know ! 
thanks guys !

**Comment by VFansss** [comment]
The 1.8.1 version of the fix should be perfectly compatible with the fix, as "v1.8.1 changelog" explain.

However Is not implementing dgVodoo (yet).

Shoudn't be a good achievement, for now.

**Comment by Tomasak** [comment]
Great new! However i checked it and it still drops quartz.dll into game folder, even it's gog release. Isn't that used to fix the green screen on tanker chapter? Extra reports im using gog version tho.

**Comment by VFansss** [comment]
You are right. I'm fixing it right now and releasing a new version with another minor fix.

**Comment by VFansss** [comment]
Yeah, I have to say that you are right.

It should be basically influent, but I've updated the 1.8.1 version (I preferred to update that version instead of releasing a new one, for various reasons) of the fix to avoid deploying it if using the GOG version.

Beware that it will not delete quartz.dll in your case, where is already deployed
